### DISCUSSION TITLE
Why the DNNConnect.CKEditorProvider.FileBrowser is missing a searchbox?

### DISCUSSION THREAD

**Discussion by edisfera** [dnnsoftware/Dnn.Platform#5566] [General]
Why the DNNConnect.CKEditorProvider.FileBrowser is missing a searchbox such as the one that is provided in the Resource Manager? When adding a link to an existing file on the server and there are many of them, the provided options (Detail View, List View, Icons View, Sort by name ascending and descending, Sort by date ascending and descending) are not enough to quickly find a file... one has to randomly navigate pages to find the initial letter of the filename and then after to find the correct page where the file is supposed to be.
Thanks

**Comment by valadas** [comment]
I gues simply because nobody coded that in...
